client/src/views/Authentication/AuthLogin.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="row register-container">
```

Comment:
Why do you declared registed-container class if your component is AuthLogin.vue, probably would be better to rename to auth-container.